.github/workflows/ci.yml

```diff
@@ -48,6 +48,13 @@ jobs:
           PLANTUML_EXEC: java -Djava.awt.headless=true -jar ${{github.workspace}}/plantuml.jar
         run: |
           py.test --color=yes tests/
+      - name: Archive failures
+        uses: actions/upload-artifact@v2
```

Comment:
I’d prefer not to, just `cat` those files in the build script, or ensure enough of them is shown by pytest.